docs/usage.md

```diff
@@ -355,8 +351,8 @@ Triage crashes found by AFL++
                                    debug]
       -j, --jobs <jobs>            Number of parallel jobs for generating CASR reports
                                    [default: half of cpu cores]
-      -t, --timeout <SECONDS>      Timeout (in seconds) for target execution [default:
-                                   disabled]
+      -t, --timeout <SECONDS>      Timeout (in seconds) for target execution, 0 value means
+                                   that timeout is disabled [default: 0]
```

Comment:
Maybe, we can do the similar with casr-san/casr-python, etc. tools? That `0` means disabling timeout

Reply:
done